Sources/PayPalNativeCheckout/Extensions/Environment+Extension.swift

```diff
@@ -1,4 +1,4 @@
-@_implementationOnly import PayPalCheckout
+import PayPalCheckout
```

Comment:
Were these moved to `import` instead of `@_implementationOnly` because of how we are integrating the SDK now? Just curious if that's the reasoning for the change or something else I am missing.

Reply:
yep previously we were not exposing callbacks, now were doing so it can't be `@_implementationOnly`